src/common/mfu_daos.c

```diff
@@ -1430,17 +1427,18 @@ static int serialize_recx_array(struct hdf5_args *hdf5,
         memset(eprs, 0, sizeof(eprs));
 
         /* list all recx for this dkey/akey */
-            number = 5;
+        number = 5;
         rc = daos_obj_list_recx(*oh, DAOS_TX_NONE, dkey,
                                 akey, &size, &number, recxs, eprs, &recx_anchor,
                                 true, NULL);
-        printf("RECX RETURNED: %d\n", (int)number);
-        printf("RECX SIZE: %d\n", (int)size);
+        if (rc != 0) {
+            MFU_LOG(MFU_LOG_ERR, "Failed to list recx: %d", rc);
```

Comment:
@dsikich Formatted -DER
```suggestion
            MFU_LOG(MFU_LOG_ERR, "Failed to list recx "DF_RC, DP_RC(rc));
```